src/static/_redirects

```diff
@@ -0,0 +1,44 @@
+# Netlify Redirect Rules
+# https://www.netlify.com/docs/redirects/
+
+# Redirect default Netlify subdomain to primary domain
+https://eslint.netlify.com/* https://eslint.org/:splat 301!
+
+# External Redirects
+/cla                                https://cla.js.foundation/eslint/eslint 302!
+/conduct                            https://code-of-conduct.openjsf.org/ 302!
+/chat/tsc-meetings                  https://discord.gg/3brqTzJ 302!
+/chat/help                          https://discord.gg/nJdtBaF 302!
+/chat                               https://discord.gg/8szcydm 302!
+
+# Internal Redirects
+/docs                               https://eslint.org/ 301!
+/docs/user-guide/rules              https://eslint.org/docs/rules 301!
+/docs/rules/id-blacklist            https://eslint.org/docs/rules/id-denylist 301!
+
+/docs/0.24.1/command-line-interface https://eslint.org/docs/user-guide/command-line-interface 301!
+/docs/0.24.1/configuring            https://eslint.org/docs/user-guide/configuring 301!
```

Comment:
I believe we don't need these internal redirects for docs since we will have a separate website for docs.

/cc @nzakas can you confirm?

Reply:
> I believe we don't need these internal redirects for docs since we will have a separate website for docs.

Thought we might later on update the links. For now wanted to add proper working links

Reply:
For now all we need is /docs/* redirecting to eslint.org/docs/*

Later we can change to docs.eslint.org/*

Reply:
done